### ISSUE TITLE
restoring from gbck

### ISSUE THREAD

**Issue by ika-musuko** [jukben/gbck#2]
this tool is so good i love it

let's say i have a new machine or something and i want to restore my files. of course i could just clone my original repo but does gbck support restoring the files to their correct locations from the config file? and if so how does it work?

**Comment by jukben** [comment]
Hi @ika-musuko,

thank you for such a lovely feedback! I have to admit I'm not reinstalling the computer that often, BUT I totally understand your needs. The situation right now is that it's not possible. But it should be pretty doable. I would suggest a new command *restore*.

Example:
```bash
gbck restore <repository>
```

First, this would ask you for a name for the project.

Next, it would restore *only* files (not commands' stdouts). If the file will exist it'd ask you if you want to overwrite it or now.

However, this will be possible just for "backups" with `gbck` settings versioned as well.

Does it make a sense? 👍

**Comment by ika-musuko** [comment]
yeah i like that syntax! and yeah that's how i envisioned this new flag working.  i especially think prompting for overwrite is a good idea lol
